caluma/caluma_form/factories.py

```diff
@@ -1,3 +1,4 @@
+import faker
 from factory import Faker, LazyAttribute, Maybe, SubFactory, lazy_attribute
```

Comment:
Do we still need `Faker` here? It seems like we're importing it kind of twice.

Reply:
`Faker` uses `faker.Faker` internally, but works only for field declarations in factories since the recent version.
So `Faker` is still used for declared fields, while we need `faker.Faker` for generating fake values manually at runtime.

Reply:
:+1: